src/main/java/com/cairn/waypoint/dashboard/service/helper/HomeworkQuestionHelperService.java

```diff
@@ -14,6 +21,32 @@ public HomeworkQuestionHelperService(ExpectedResponseDataService expectedRespons
     this.expectedResponseDataService = expectedResponseDataService;
   }
 
+  public HomeworkQuestionDto generateHomeworkQuestionDto(HomeworkQuestion question) {
+	  return HomeworkQuestionDto.builder()
```

Comment:
Again note your use of the builder paradigm when a mapper could do this for you.